lib/commands/project.js

```diff
@@ -65,6 +65,81 @@ function sortImages(images) {
   return images;
 }
 
+project.resize = function (projectName, servoSize, cb) {
+  var projectId, projectName;
+
+  projectController.find({
+      userId : userConfig.data.userId
+    },
+    function (err, projects) {
+      if (err) {
+        err = error.handleApiError(err, 'GET_PROJECTS', cb);
+        if (err.length > 0) {
+          return cb(err);
+        }
+      }
+      if (projects.length === 0) {
+        modulus.io.error('You currently have no projects. One can be created using the create command.');
+        return cb();
+      }
+
+      async.series([
+        function promptForProject(fn) {
+          project.chooseProjectPrompt(projects, projectName, function (err, result) {
+            if (err) {
+              return fn(err);
+            }
+
+            if (!result) {
+              return fn('You must select a project.');
+            }
+
+            projectId = result.id;
+            projectName = result.name;
+            projectDomain = result.domain;
+
+            fn();
+          });
+        },
+        function promptForServoSize(fn) {
+
+          function handleServoSize(err, result) {
```

Comment:
I like this function. I can see it replacing other things in this file, but with a little more work. Maybe have it return `true`/`false` instead of handling a callback and only validate that the `servoSize` is a valid one.